helm/cas-airflow/Chart.yaml

```
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
```

```suggestion
apiVersion: v2
```
https://helm.sh/docs/topics/charts/#the-apiversion-field